partner_priority/data/partner_priority_data.xml

```diff
@@ -0,0 +1,25 @@
+<!-- Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo noupdate="1">

```

Comment:
If you have priorities like 1 = Low,  3 = Normal, 5 = Elevated, 8 = Critical, you make it easier to add intermediate priorities, if somebody wants a more fine grained categorization.

Reply:
I changed the labels and sequence numbers on the imported data.